src/services/export/XLSExporterForObservations.ts

```diff
@@ -1,20 +1,86 @@
-import path from 'path';
+import stream from 'stream';
 import { NextFunction, Request, Response } from 'express';
-// import { getRepository, Repository } from 'typeorm';
+import { getRepository, Repository } from 'typeorm';
+import { write, utils } from 'xlsx';
 import AbstractExporter, { ExporterType } from './AbstractExporter';
-// import { Observation } from '../../entities/observation-entity';
+import { Observation } from '../../entities/observation-entity';
 
 export default class XLSExporterForObservations extends AbstractExporter {
   public type: ExporterType = 'XLS';
 
   public route: string = 'observations';
 
-  // private observations: Repository<Observation> = getRepository(Observation);
+  private observations: Repository<Observation> = getRepository(Observation);
+
+  private ObservationColumns = [
+    'ringMentioned',
+    'photos',
+    'distance',
+    'direction',
+    'elapsedTime',
+    'colorRing',
+    'date',
+    'geographicalCoordinates',
```

Comment:
It is outdated field. Could you please use latitude and longitude instead?

Reply:
already done)